packages/node/src/Client.ts

```diff
@@ -77,6 +80,19 @@ class Client {
   track(event: string, uid: string, data: Record<string, any>): Promise<Response> {
     return this.makeHttpRequest('track', { event, uid, data })
   }
+
+  content(page: string, options?: ContentOptions) {
```

Comment:
```
content(
  contentType: ContentOptions['contentType'],
  options?: Omit<ContentOptions, 'contentType'>
)
```

Reply:
Removing `contentType` from `ContentOptions`, it'll save us from omits and union types.